marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -77,7 +79,7 @@ deposit :: (
     -> m MarloweData
 deposit tx marloweData accountId amount = do
     pubKey <- ownPubKey
-    applyInputs tx marloweData [IDeposit accountId pubKey amount]
+    applyInputs tx marloweData [IDeposit accountId pubKey Ada.adaSymbol Ada.adaToken (Ada.getLovelace amount)]
```

Comment:
do we need to use qualified `Ada.` everywhere since the function names already have `ada` in them?